src/MetaModels/Events/DefaultPropertyConditionCreator.php

```diff
@@ -70,6 +72,22 @@ public function handle(CreatePropertyConditionEvent $event)
                     $meta['value']
                 ));
                 break;
+            case 'conditionpropertymultiplevalueis':
```

Comment:
When we add another condition, we DEFINATELY will be in need for a factory here as well (see TODO comment).
Maybe you want to give it a try to implement? :)